src/+otp/Problem.m

```diff
@@ -242,8 +242,8 @@ function validateNewState(obj, newTimeSpan, newY0, newParameters)
         end
         
         function mov = internalMovie(obj, t, y, varargin)
-            mov = otp.utils.movie.TrajectoryMovie(obj.Name, @obj.index2label, varargin{:});
-            mov.record(t, y);
+            mov = otp.utils.movie.PhasePlaneMovie(obj.Name, @obj.index2label, varargin{:});
+            mov.record(t, y(1:2,:));
```

Comment:
Looks like this code was used for a quick test and got committed accidentally

Reply:
The first two variables of the state are the relevant ones for the movie here. Unlike plot, I don't think Movie functions support indexing( for now). So that why I only pass in the trajectory of y1 and y2.

Reply:
This is the `Problem` class not `ArenstorfProblem`.  This breaks all the other problem movies and needs to be reverted.

Reply:
whoops :)

Reply:
Haha.